src/test/java/com/epam/tc/hw7/pages/JdiSite.java

```diff
@@ -0,0 +1,42 @@
+package com.epam.tc.hw7.pages;
+
+import com.epam.jdi.light.elements.complex.Menu;
+import com.epam.jdi.light.elements.pageobjects.annotations.JSite;
+import com.epam.jdi.light.elements.pageobjects.annotations.locators.Css;
+import com.epam.jdi.light.elements.pageobjects.annotations.locators.UI;
+import com.epam.jdi.light.ui.html.elements.common.Button;
+import com.epam.jdi.light.ui.html.elements.common.Icon;
+import com.epam.tc.hw7.model.User;
+import io.qameta.allure.Step;
+
+@JSite("https://jdi-testing.github.io/jdi-light/")
+public class JdiSite {
+    public static HomePage homePage;
+    public static MetalsAndColorsPage metalsAndColorsPage;
+    @UI(".m-l8 li")
```

Comment:
In JdiSite class need to declare variable WebPage class

Reply:
Fixed. Now jdiSite contains only parts of webpage declaration, all fields and methods are placed in homePage metalsAndColorsPage